src/client/Client.cc

```diff
@@ -14137,9 +14137,9 @@ int Client::_rename(Inode *fromdir, const char *fromname, Inode *todir, const ch
   }
```

Comment:
@rishabh-d-dave 

Good catch. And could you add one comments here ?

`// don't allow cross-quota renames`

Reply:
Sure.

Reply:
On second thought, sounds little vague to me. I've added it but can you suggest something that offers better explanation?

Reply:
Currently we do not allow to move a file or directory into a new path, which has a different quota realm. IMO we can support this in theory but the code maybe a bit complicated, currently both the kclient and userspace just disable cross-quota renames.